content/events/anvil2023-january-office-hours.md

```diff
@@ -0,0 +1,30 @@
+---
+author: "AnVIL"
+conference: "AnVIL 2023"
+description: "Helping users find solutions to their AnVIL questions."
+eventType: "Office Hours"
+featured: true
+location: "Virtual"
+sessions: [{sessionStart: "26 Jan 2023 11:00 AM"}]
+timezone: "America/New_York"
+title: "AnVIL Office Hours - February 2023"
+---
+
+<event-hero></event-hero>
+
+### Description
+
+These Office Hours are a new opportunity for you to get your questions about working on AnVIL answered in person –
```

Comment:
```suggestion
These Office Hours are an opportunity for you to get your questions about working on AnVIL answered in person –
```

Reply:
@jpaten I think the `Description` text in the ticket has been updated since previous Office Hours... would you mind updating as per the ticket text?